Update contribute page (#1436)
act-rules/act-rules.github.io: pages/contribute/join.md

---
title: Join the Community Group
---

The best way to stay informed about the activities of ACT-Rules Community is to become a member.

## Mailing list keeps you up to date

> By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if no feedback

Reviewer: This sentence feels unfinished.

Reviewer 2: Perhaps...

```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if there is no unresolved feedback.
```

> By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if no feedback

Reviewer 2: Perhaps...

```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if there is no unresolved feedback.
```

> By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if no feedback

Reviewer 3: I'm not sure about the reference to Github here. As in, it might scare people away feeling they need to know what is Github and what is a pull request in order to join the CG.
```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes (new rules, correction to existing ones, …) that have been approved by at least 3 other members, and will be published once there is no unresolved feedback.
```

Maybe also add a rough estimate of the number of emails (2-3/week?) to avoid scaring people in fear of being "spammed" (the list here sounds like there is a lot happening on the ML, but it is not really the case).

Reviewer 2: That's a good suggestion. Although it may still be worth mentioning that those announced changes may be viewed on the Github without needing an account (which I think it true).

## Join us on Slack

The ACT-Rules community is active in the #act-rules channel on the W3C Community Slack. To join, you will need to [sign up for the W3C community Slack](https://w3ccommunity.slack.com/join/shared_invite/zt-drq1eer6-MlZgOJOWVAEV1UTp66ywAQ#/). Once you have signed up, you can join by visiting [w3ccommunity.slack.com](https://w3ccommunity.slack.com/).

## Participate in our calls

> The Community Group has conference calls twice a month. The first call is (usually) on the **second Thursday of the month** from [16:00 to 17:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200813T16&p1=16&ah=1&am=00). The second monthly call is on every **fourth Thursday of the month** from [10:00 to 11:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200827T10&p1=16&ah=1&am=00).

Reviewer 2: ```suggestion
The Community Group has conference calls twice a month. Usually, the first call is on the **second Thursday of the month** from [16:00 to 17:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200813T16&p1=16&ah=1&am=00), and the second monthly call is on every **fourth Thursday of the month** from [10:00 to 11:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200827T10&p1=16&ah=1&am=00).
```
